src/composables/transaction.ts

```diff
@@ -22,22 +24,22 @@ interface ITokenTransactionComposable extends ITx {
   isAe: string
 }
 
-export const useTransactionToken = (
-  initTransaction: ITransaction | undefined,
+export const useTransaction = (
```

Comment:
I noticed that the composable options are not passed in a most common way: as an object. Could you please align it?

```
interface UseTransactionOptions {
  showDetailedAllowanceInfo?: boolean
}

export function ({
  showDetailedAllowanceInfo = false,
}: UseTransactionOptions = {}) {
```